modules/post/windows/gather/enum_av_excluded.rb

```diff
@@ -0,0 +1,86 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'rex'
+
+class Metasploit3 < Msf::Post
+
+  include Msf::Post::Windows::Registry
+
+  def initialize(info={})
+    super( update_info( info,
+        'Name'          => 'Windows Antivirus Excluded Locations Enumeration',
+        'Description'   => %q{ This module will enumerate all excluded directories within supported AV products },
+        'License'       => MSF_LICENSE,
+        'Author'        => [ 'Andrew Smith'],
+        'Platform'      => [ 'win' ],
+        'SessionTypes'  => [ 'meterpreter' ]
+      ))
+
+  end
+
+  def excluded_list
+    if registry_enumkeys("HKLM\\SOFTWARE\\Microsoft").include?("Microsoft Antimalware")
```

Comment:
Just because the keys exist does not necessarily mean that the application in question is enabled.  I think this may be relevant in cases where Essentials was installed and configured but then SEP was installed which disables Essentials.  Your code seems to only print one, but perhaps it should print both.